src/ui-lib/sass/01-tools/_container.scss

```diff
@@ -2,6 +2,10 @@
 @mixin grav-l-container($no-margin: false) {
   max-width: $grav-page-content-max-width;
 
```

Comment:
please add this link in a comment on why this is a bug: https://alastairc.ac/2014/08/flexbox-ie11-bugs/ and add also the link to OUR bug report